server/heroku/processes.go

```diff
@@ -91,6 +92,15 @@ func (h *PostProcess) ServeHTTPContext(ctx context.Context, w http.ResponseWrite
 
 		fmt.Fprintf(outStream, "HTTP/1.1 200 OK\r\nContent-Type: application/vnd.empire.raw-stream\r\n\r\n")
 
+		// We send the null character periodically, to keep the connection alive.
+		// Otherwise the connection would close after the ELB idle connection timeout.
+		go func() {
+			for {
+				fmt.Fprintf(outStream, "\x00")
```

Comment:
Yeah, that's what I put on the PR preview (minus the stop which is a good pt). I'll update.